website/docs/plugin/framework/migrating/resources/schema.mdx

```diff
@@ -0,0 +1,224 @@
+	}
+	...
+},
+```
+
+SDKv2 defines the `schema.Resource` struct as follows.
+
+```go
+schema.Resource{
+	Schema 			map[string]*Schema
+	SchemaVersion 		    int
+	MigrateState 			StateMigrateFunc
+	StateUpgraders 		    []StateUpgrader
+	Create 			        CreateFunc
+	Read 				    ReadFunc
+	Update 			        UpdateFunc
+	Delete 				    DeleteFunc
+	Exists 				    ExistsFunc
+	CreateContext 		    CreateContextFunc
+	ReadContext 			ReadContextFunc
+	UpdateContext 		    UpdateContextFunc
+	DeleteContext 		    DeleteContextFunc
+	CreateWithoutTimeout	CreateContextFunc
+	ReadWithoutTimeout 		ReadContextFunc
+	UpdateWithoutTimeout	UpdateContextFunc
+	DeleteWithoutTimeout 	DeleteContextFunc
+	CustomizeDiff 			CustomizeDiffFunc
+	Importer 			    *ResourceImporter
+	DeprecationMessage 		string
+	Timeouts 			    *ResourceTimeout
+	Description 			string
+	UseJSONNumber 		    bool
+}
+```
+
+## Framework
+
+In the Framework, you define your provider's resources by adding them to your provider's `GetResources` function.
+
+The `GetResources` function on your `provider.Provider` returns a map from the resource name (string) to a struct that
+implements the `ResourceType` interface for each resource your provider supports.
+
+The following code shows how you add a resource to your provider with the Framework.
+
+```go
+func (p *provider) GetResources(ctx context.Context) (map[string]provider.ResourceType, diag.Diagnostics) {
+    return map[string]provider.ResourceType{
+        "resource_example": resourceTypeExample{},
+    }, nil
+}
+```
+
+The `provider.ResourceType` interface requires `GetSchema` and `NewResource` functions.
+
+The `GetSchema` function returns a `tfsdk.Schema` struct which defines your resource's attributes. This is the same
+struct you use to define provider and data source attributes.
+
+The `NewResource` function returns a type that you define. The type implements the `resource.Resource` interface,
+including the CRUD functions for your resource.
+
+The following code shows how you define a `provider.ResourceType` which implements these two functions with the
+Framework.
+
+```go
+type resourceTypeExample struct{}
+
+func (r resourceTypeExample) GetSchema(context.Context) (tfsdk.Schema, diag.Diagnostics) {
+	return tfsdk.Schema{}, nil
+}
+
+func (r resourceTypeExample) NewResource(_ context.Context, p provider.Provider) (resource.Resource, diag.Diagnostics) {
+	return &resourceExample{}, nil
+}
+```
+
+The `tfsdk.Schema` is used for defining the schema for providers, resources and data sources. The Framework defines the
+`tfsdk.Schema` struct as follows.
+
+```go
+type Schema struct {
+	Attributes map[string]Attribute
+	Blocks map[string]Block
+	Version int64
+	DeprecationMessage  string
+	Description         string
+	MarkdownDescription string
+}
+```
+
+Refer to the [Resources - CRUD functions](FIXME: link) page in this guide to learn how to define the `resource.Resource`
+type returned by your resource type's `NewResource` function.
```

Comment:
don't forget the link!

Reply:
Have added a link and re-worded slightly.